src/components/Footer.scss

```diff
@@ -1,9 +1,9 @@
-.Footer {
-  height: 1rem;
-  composes: Montserrat from '../styles/fonts.scss';
```

Comment:
As we talked on slack we shouldn't use `composes` feature of css-modules. Same goes for Header.scss file